src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
*Error*?

what if `people` variable is `undefined`, `null` or even a [primitive](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) value?

We should have this issue in mind to prevent unwanted behaviors.